utils/vim/README.md

```diff
@@ -0,0 +1,29 @@
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+# Carbon Syntax Highlighting for Vim & Neovim
+
+For Carbon devs using Vim or Neovim, this plugin provides syntax highlighting for .carbon files found throughout `explorer/testdata`
```

Comment:
Note running this through prettier or pre-commit will do word-wrap for you.

https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/contribution_tools.md#pre-commit
https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/contribution_tools.md#prettier